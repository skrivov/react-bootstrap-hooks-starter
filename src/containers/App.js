import React from "react";
import Container from "react-bootstrap/Container";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";

import Header from "../components/Header";
import Speakers from "./Speakers";
import Sessions from "./Sessions";
import "./App.css";
export const ConfigContext = React.createContext();

const App = () => {
  return (
    <Container>
      <Header />
      <Router>
       
          <Switch>
            <Route path="/sessions">
              <Sessions />
            </Route>
            <Route path="/speakers">
              <Speakers />
            </Route>
            <Route path="/">
              <Sessions />
            </Route>
          </Switch>
     
      </Router>
    </Container>
  );
}

export default App;
