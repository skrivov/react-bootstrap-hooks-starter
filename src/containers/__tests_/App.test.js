import React from "react";
import App from "../App";
import { shallow } from "enzyme";

import { create, act } from "react-test-renderer";

it("renders without crashing", () => {
  shallow(<App />);
  // render the component
  let root;
  act(() => {
    root = create(<App value={1} />);
  });

  // make assertions on root
  expect(root.toJSON()).toMatchSnapshot();
});
