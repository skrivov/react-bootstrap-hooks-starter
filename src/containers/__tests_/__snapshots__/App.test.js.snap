// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1>
      Software Design in the Era of Cloud
    </h1>
    <p>
      Discussing New Trends, Patterns , and  Best Practices
    </p>
  </div>
  <nav
    className="navbar navbar-expand navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      Enterprise Academy 
    </a>
    <div
      className="navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="nav-link"
        data-rb-event-key="/"
        href="/"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        Sessions
      </a>
      <a
        className="nav-link"
        data-rb-event-key="/speakers"
        href="/speakers"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        Speakers
      </a>
    </div>
  </nav>
  <div>
    Loading...
  </div>
</div>
`;
